emperor/core.py

```diff
@@ -514,14 +587,26 @@ def _process_data(self, custom_axes, jackknifing_method):
         if low is not None or high is not None:
             ci = np.abs(high - low).tolist()
 
+        if self.ordination.features is not None:
+            bi_coords = self.ordination.features.values[:, :dims]
+            bi_coords /= np.max(np.abs(bi_coords))
+            bi_coords = bi_coords.tolist()
+            bi_ids = self.ordination.features.index.values.tolist()
+
+            bi_headers, bi_metadata = self._to_legacy_map(self.feature_mf)
+
         return (self.ordination.samples.index.tolist(), c_data.tolist(),
-                c_pct, ci, headers, metadata, names)
+                c_pct, ci, headers, metadata, names, bi_coords, bi_ids,
+                bi_headers, bi_metadata)
 
-    def _to_legacy_map(self, custom_axes=None):
+    def _to_legacy_map(self, mf, custom_axes=None):
         """Helper method to convert Pandas dataframe to legacy QIIME structure
 
         Parameters
         ----------
+        mf : pd.DataFrame
+            DataFrame with the metadata, this can be feature or sample
```

Comment:
Does the index matter for this dataframe?

Reply:
It doesn't really matter, I've expanded on the description.